src/lib/Form/Type/User/ChangePasswordType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Type\User;
+
+use EzSystems\EzPlatformAdminUi\Form\Data\User\ChangePasswordData;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ChangePasswordType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('oldPassword', PasswordType::class, [
+                'required' => true,
+                'label' => /** @Desc("Old password") */ 'ezplatform.change_password.old_password',
+            ])
+            ->add('newPassword', RepeatedType::class, [
+                'type' => PasswordType::class,
+                'invalid_message' => /** @Desc("The password fields must match.") */ 'ezplatform.change_password.passwords_must_match',
+                'required' => true,
+                'first_options' => ['label' => /** @Desc("New password") */ 'ezplatform.change_password.new_password'],
+                'second_options' => ['label' => /** @Desc("Repeat new password") */ 'ezplatform.change_password.repeat_new_password'],
+            ])
+            ->add(
+                'save',
```

Comment:
It's advised to have submit button named after the action it's performing. In this case it should be `change`. Displayed name can be customized with label if need.